app/views/admin/communities/edit_welcome_email.haml

```diff
@@ -34,7 +34,7 @@
   - content_for(:resend_link) do
     = link_to(t("admin.communities.outgoing_email.resend_link"), nil, class: "js-sender-address-resend-verification")
 
-  - if user_defined_address.nil?
+  - if user_defined_address.nil? || !ses_in_use
```

Comment:
I'm not sure what the intent is in `user_defined_address.nil?`, so I was not sure if I should remove it or not.